README.md

```diff
@@ -5,7 +5,7 @@ This repository contains the scripts and pre-compiled binaries used to create th
 Please check the Raspberry Pi [general discussion forum](https://www.raspberrypi.org/forums/viewforum.php?f=63) if you have a support question. 
 
 # Reset to factory defaults
-The [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) provides GUI for creating a GUI to update the bootloader and configure the default boot-mode.
+[Raspberry Pi Imager](https://www.raspberrypi.org/downloads/) can be used to restore your Raspberry Pi 4 EEPROMs to the factory defaults: the current default EEPROM images will be installed, and the configuration reset to factory defaults. Choose `misc utility images` from the list of operating systems, then `bootloader`, then select the desired boot order.
```

Comment:
"the current default EEPROM images" sounds a bit weird? :shrug: Perhaps using backticks works better? "the current `default` EEPROM images". Although I'm still not sure if there's any benefit in explicitly specifying "EEPROM images" rather than just "bootloader"? I'm happy to defer to @timg236 's decision on this.

Please use the correct capitalisation on the `Misc utility images` and `Bootloader` labels.